.gitigonre

```diff
@@ -0,0 +1,19 @@
+# Created by https://www.toptal.com/developers/gitignore/api/web
+# Edit at https://www.toptal.com/developers/gitignore?templates=web
+
+### Web ###
+*.asp
+*.cer
+*.csr
+*.css
+*.htm
+*.html
```

Comment:
Most of these are reasonable source files, why ignore them and not node_modules?

Reply:
That's correct, they are reasonable source files.
I have used [gitignore.io](https://www.toptal.com/developers/gitignore) to generate a default `.gitignore`  file.
Should it only include `node_modules/`?